canonical/ubuntu.com
# Rebrand /iot/smart-displays

## Done

- Apply rebranding on /internet-of-things/smart-displays
- No changes to content, only design

## QA

- Go to https://ubuntu-com-14538.demos.haus/internet-of-things/smart-displays
- Check that design matches [Figma](https://www.figma.com/design/1CdIT6TV3ASf55GsnQcwXg/Internet-of-Things?node-id=2001-15688&node-type=frame&m=dev)
- No changes to design

## Issue / Card

Fixes [WD-16639](https://warthogs.atlassian.net/browse/WD-16639)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-16639]: https://warthogs.atlassian.net/browse/WD-16639?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
